examples/src/main/java/com/mapbox/navigation/examples/core/MapboxNavigationActivity.kt

```diff
@@ -156,6 +157,22 @@ class MapboxNavigationActivity : AppCompatActivity() {
             }
         }
 
+    private val maneuverCallback = ManeuverCallback { maneuvers ->
+        binding.maneuverView.renderManeuver(maneuvers)
+        maneuvers.fold(
+            { error ->
+                Toast.makeText(
+                    this@MapboxNavigationActivity,
+                    error.errorMessage,
+                    Toast.LENGTH_SHORT
+                ).show()
+            },
+            { list ->
+                binding.maneuverView.renderUpcomingManeuvers(list.subList(1, list.size))
```

Comment:
`MapboxManeuverView` is already handling the visibility of upcoming maneuver list. When you click on the current maneuver, it shows the list. When you click on the maneuver again, it will hide the list. If I expose a method that allows you to hide the upcoming list for ever, clicking on the current maneuver will show it again. To fix this, I would have to introduce some flag that would track if user have disabled showing the upcoming list. If true, then on click on current maneuver won't work. I don't like this approach because then I have to maintain the state which would be lost the moment the device is rotated. As for `maneuvers.subList(1, maneuvers.size)` this doesn't necessarily mean that you have to know the internals of the API, because all the API tells is `renderUpcomingManeuver` starting at index 1 to the end of the list. It's upto you how many items in the list you want to display.  This API is rather advantageous, where you can decide to only display 3 upcoming instructions rather than whole list of 100 upcoming instructions as done in Google Maps. Handling this internally in the view would always render all the instructions and all you could do is either show/hide it.
cc @LukasPaczos

Reply:
I changed the signature of the method as follows:

```
/**
     * Invoke the method to render list of upcoming instructions on top of [MapboxManeuverView]
     * starting at fromIndex and ending at toIndex. If not specified, the fromIndex is 1 as maneuver
     * at index 0 is displayed as current maneuver and endIndex is the last index in the list.
     * @param maneuvers List<Maneuver>
     * @param fromIndex Int The starting index of the list
     * @param toIndex Int The end index of the list
     */
    fun renderUpcomingManeuvers(
        maneuvers: List<Maneuver>,
        fromIndex: Int = 1,
        toIndex: Int = maneuvers.size
    )
```

Reply:
> If I expose a method that allows you to hide the upcoming list for ever, clicking on the current maneuver will show it again.

We already do expose that method in
```kotlin
fun updateUpcomingManeuversVisibility(visibility: Int)
```
unless it does something different?

I personally see the `ManeuverView` as a single entity where you provide a list of maneuvers and the first one is the current one shown at the main banner and the rest is the overflow that is visible after expanding the list (or scrolling horizontally in the future). You can still manage the number of "upcoming" maneuvers shown by trimming the list if you'd like to (same as you would do with a dedicated method), and it would simplify the API to a single method that accepts a list that developers can curate if they like instead of having to consider which method to invoke, they have the same signature after all.